torch/csrc/jit/passes/onnx/shape_type_inference.h

```diff
@@ -82,5 +82,7 @@ void UpdateReliable(
     torch::jit::Value* output,
     const std::pair<bool, bool>& input_reliable);
 
+void UpdateReliable(torch::jit::Node* n);
```

Comment:
Just wondering: Do we need this to be in the header?

Reply:
Yes, there is another one taking node->output as params, and this is defined after ProcessConstantValueMap.